app/Http/Controllers/MessagesController.php

```diff
@@ -2,20 +2,37 @@
 
 namespace Gladiator\Http\Controllers;
 
+use Gladiator\Models\Contest;
 use Gladiator\Models\Message;
 use Gladiator\Events\QueueMessageRequest;
 
 class MessagesController extends Controller
 {
     /**
+     * Display the specified resource.
+     *
+     * @param  string  $id  Contest ID
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $messages = Message::where('contest_id', '=', $id)->get();
+
+        return view('messages.show', compact('messages'));
+    }
```

Comment:
Won't conflict, but I currently am working on a view for showing all the messages and editing them within a single collected view after clicking a button on the Contests page to "View & Edit Messages". Just noting here that we may want to consolidate.